data/src/main/java/com/microsoft/azure/kusto/data/StringUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.azure.kusto.data;
+
+public class StringUtils
+{
+    public static boolean startsWithIgnoreCase(String str, String prefix)
+    {
+        return str.regionMatches(true, 0, prefix, 0, prefix.length());
+    }
+
+    public static boolean endsWithIgnoreCase(String str, String suffix)
```

Comment:
Both of these are available in import org.apache.commons.lang3.StringUtils; which we use elsewhere in the code

Reply:
You resolved this, but the code is still here